Fix docs(91-100)

longranger2 opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->
Others 
### PR changes
<!-- One of [ OPs | APIs | Docs | Others ] -->
Docs
### Describe
<!-- Describe what this PR does -->
1. 修改linspace参数"stop"的介绍
2. 更新了conv2d_transpose英文文档中 1.公式异常 2.参数说明异常 3.padding=SAME和VALID的公式说明

对应的中文文档修改 https://github.com/PaddlePaddle/docs/pull/5504

longranger2 committed:
修改linspace参数"stop"的介绍

longranger2 committed:
更新了conv2d_transpose英文文档中 1.公式异常 2.参数说明异常 3.padding=SAME和VALID的公式说明; test=docs_preview

longranger2 committed:
Merge branch 'PaddlePaddle:develop' into develop

Ligoml commented on python/paddle/tensor/creation.py:
```diff
@@ -272,12 +272,12 @@ def create_tensor(dtype, name=None, persistable=False):
         start(int|float|Tensor): The input :attr:`start` is start of range. It is a int, float, \
             or a 0-D Tensor with data type int32, int64, float32 or float64.
-        stop(int|float|Tensor): The input :attr:`stop` is start variable of range. It is a int, float, \
+        stop(int|float|Tensor): The input :attr:`stop` is end variable of range. It is a int, float, \
```
```suggestion
        stop(int|float|Tensor): The input :attr:`stop` is end of range. It is a int, float, \
```

longranger2 committed:
解决了
1.英文版Note中的api_fluid_layers_conv2d的链接失效，
2. 英文版参数中的padding_start的参数错误
3. 参数中bias_attr和param_attr的链接失效
4. 参数缺少optional

longranger2 committed:
更新了paddle.static.auc的英文文档的Return描述以及函数的参数

longranger2 committed:
Update python/paddle/tensor/creation.py

Co-authored-by: Ligoml <39876205+Ligoml@users.noreply.github.com>

longranger2 committed:
更新了shard_index的ignore_value描述

longranger2 committed:
Merge remote-tracking branch 'origin/develop' into loneranger_paddle

longranger2 committed:
修改了英文文档的paddle.static.nn.conv3d_transpose的公式介绍

Ligoml commented on python/paddle/static/nn/common.py:
```diff
@@ -1417,24 +1427,40 @@ def conv3d_transpose(
-           D^\prime_{out} &= (D_{in} - 1) * strides[0] - 2 * paddings[0] + dilations[0] * (D_f - 1) + 1 \\
-           H^\prime_{out} &= (H_{in} - 1) * strides[1] - 2 * paddings[1] + dilations[1] * (H_f - 1) + 1 \\
-           W^\prime_{out} &= (W_{in} - 1) * strides[2] - 2 * paddings[2] + dilations[2] * (W_f - 1) + 1 \\
+           D^\prime_{out} &= (D_{in} - 1) * strides[0] - pad_depth_front - pad_depth_back + dilations[0] * (D_f - 1) + 1 \\
```
```suggestion
           D^\prime_{out} &= (D_{in} - 1) * strides[0] - pad_{depth_front} - pad_{depth_back} + dilations[0] * (D_f - 1) + 1 \\
```

Ligoml replied:
另外这个公式好像不能改吧？这是conv2d_transpose的公式？

Ligoml replied:
这里的公式部分中英文文档是不一致的，@jerrywgz 看下哪边是对的？

longranger2 replied:
公式是conv3d_transpose的公式，公式部分的中英文文档确实是不一致

Ligoml replied:
咨询了一下api负责人，conv2d_transpose和conv3d_transpose是一样的情况，都是以英文文档为准，改一下中文文档吧

longranger2 replied:
好的 👌

Ligoml commented on python/paddle/tensor/manipulation.py:
```diff
@@ -568,7 +568,7 @@ def shard_index(input, index_num, nshards, shard_id, ignore_value=-1):
         nshards (int): The number of shards.
         shard_id (int): The index of the current shard.
-        ignore_value (int): An integer value out of sharded index range.
+        ignore_value (int): An integer value out of sharded index range. The default value is -1.
```
```suggestion
        ignore_value (int, optional): An integer value out of sharded index range. The default value is -1.
```

Ligoml committed:
add py_func COPY-FROM label; test=document_fix

Ligoml commented:
paddle.static.py_func 我已经改了，剩下的问题自己改一下吧~ @longranger2

longranger2 committed:
Update python/paddle/tensor/manipulation.py

Co-authored-by: Ligoml <39876205+Ligoml@users.noreply.github.com>

Ligoml committed:
formula; test=document_fix

Ligoml reviewed (dismissed):
good job！LGTM for docs
检查预览后合入

Ligoml committed:
formula; test=document_fix

Ligoml committed:
formula; test=document_fix
